apps/web/src/app/_components/onboarding/post-onboarding/welcome-dialog.tsx

```diff
@@ -0,0 +1,30 @@
+import Image from "next/image";
+
+import { Button } from "@cooper/ui/button";
+
+interface WelcomeDialogProps {
+  heading: string;
+  subheading: string;
+  buttonText: string;
+  onClick: () => void;
+}
+
+export function WelcomeDialog({
+  heading,
+  subheading,
+  buttonText,
+  onClick,
```

Comment:
So would it be crazy to pass a button as a prop rather than text and onClick. Then maybe if we are feeling lazy we can just reuse the new-review-dialog for the meantime